src/browser.ts

```diff
@@ -101,6 +132,20 @@ export class GrpcClient {
     );
   }
 
+  /**
+   * Browser version of createStub
+   * Creates a gRPC-fallback stub and auth
```

Comment:
I'm a little confused about what 'creating auth' means over here... aren't we using the OAuth2Client created elsewhere to authenticate our fetch requests? Or am I misunderstanding what this is conveying?

Reply:
Yeah, I'm guessing 'creating auth' needs to be replaced with `building the authentication header using the provided OAuth2Client instance` or something like that.